docs/ref-manual/cvl/types.md

```diff
@@ -89,6 +92,94 @@ multi-dimensional arrays
 reference types
 ```
 
+Types Defined in Solidity Contracts
```

Comment:
The above is already a section about types taken from Solidity, so I wouldn't start a new top-level section.  I might move the subsections of this section (enums / structs / value types) to between arrays and methods.

Reply:
so where do I put the bit about importing contracts to access this class of types?